### PR TITLE
Show consistent header across apps

### PR DESCRIPTION
**Pull request by jfeigel**

### CODE REVIEW

File: app/styles/app.styl [hunk 1]
```diff
@@ -22,6 +22,22 @@
...
+				font-weight 500
+				color $text
+				background lighten(black, 95%)
+				border-top 1px solid lighten($base, 90%)
+				border-bottom 1px solid lighten($base, 90%)
+				&.today
+					color $primary
+				+ .item
+					border-top 0 none
+			&.today .itemContainer .date
```
Comment (dominiksteiner) [inline app/styles/app.styl]: can you add a comment for what this rule is being used? (prepend with #2168 so we can reference this later)

Comment (jfeigel) [inline app/styles/app.styl]: @dominiksteiner I added the comment

Comment (dominiksteiner) [inline app/styles/app.styl]: Thanks, can you add another one for the display: none rule and why that has to apply? (I guess we have to hide the header for today?)

Comment (jfeigel) [inline app/styles/app.styl]: It's to hide the header on the `Today` tab

**Pull request merged**